src/components/ActiveFilters.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div>
+    <button
+      v-for="(item, key) in activeValues"
+      :key="key"
+      type="button"
+      class="active-filter btn btn-light m-1 btn-outline-secondary"
+      @click="removeFilter(item)"
+    >
+      {{ item.label }}: {{ item.value }}
+      <font-awesome-icon
+        icon="times"
+        class="ml-1"
+      />
+    </button>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue'
+
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faTimes } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+library.add(faTimes)
+
+export default Vue.extend({
+  name: 'ActiveFilters',
+  components: { FontAwesomeIcon },
+  props: {
+    filters: {
+      type: Array,
+      required: true
+    },
+    value: {
+      type: Object,
+      default: () => ({})
+    }
+  },
+  data () {
+    return {
+      isOpen: this.collapsable ? !this.collapsed : true
+    }
+  },
+  computed: {
+    activeValues () {
+      let values = []
+      Object.entries(this.value).forEach(([key, current]) => {
+        const filter = this.selectFilter(key)
+
+        // Clean op the values by removing undefined entry's
+        if (current === undefined) return
+
+        // promise function has not settled yet
+        if (typeof filter.options === 'function') return
```

Comment:
options is always function or array, if its a function returning a promise it remain a function returning a promise even if the promise is resolved, the that case the result would now not be handled i think